Dockerfile.prod

```diff
@@ -0,0 +1,71 @@
+FROM metabrainz/python:2.7
+
+ENV DOCKERIZE_VERSION v0.2.0
+RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+    && tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+                       build-essential \
+                       git \
+                       libpq-dev \
+                       libffi-dev \
+                       libssl-dev \
+                       node-less \
+                       node-clean-css \
+                       redis-tools \
+    && rm -rf /var/lib/apt/lists/*
+
+
+# This is needed to circumvent a bug in an ubunutu package
+RUN ln -s /usr/lib/python2.7/plat-*/_sysconfigdata_nd.py /usr/lib/python2.7/
+
+# PostgreSQL client
+RUN apt-key adv --keyserver ha.pool.sks-keyservers.net --recv-keys B97B0AFCAA1A47F044F244A07FCC7D46ACCC4CF8
+ENV PG_MAJOR 9.5
+RUN echo 'deb http://apt.postgresql.org/pub/repos/apt/ jessie-pgdg main' $PG_MAJOR > /etc/apt/sources.list.d/pgdg.list
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends postgresql-client-$PG_MAJOR \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN pip install uWSGI==2.0.13.1
+
+RUN mkdir /code
+WORKDIR /code
+
+# MessyBrainz
+RUN git clone https://github.com/metabrainz/messybrainz-server.git messybrainz
+WORKDIR /code/messybrainz
+RUN pip install -r requirements.txt
+RUN python setup.py install
+
+# ListenBrainz
+WORKDIR /code
+RUN git clone https://github.com/metabrainz/listenbrainz-server.git listenbrainz
```

Comment:
Is this the standard MetaBrainz way of building an image? (as opposed to running `docker build` inside a checkout and using `COPY .`)